ddpui/celeryworkers/tasks.py

```diff
@@ -762,6 +765,15 @@ def summarize_logs(
                 )
                 return
             task = dbt_tasks[0]
+            task['logs'] = []
+            limit = 0
+            while(True):
+                new_logs_set = get_flow_run_logs(flow_run_id, task_id, limit, offset=FLOW_RUN_LOGS_OFFSET_LIMIT)
+                task['logs']+=new_logs_set['logs']
+                if len(new_logs_set['logs']) == FLOW_RUN_LOGS_OFFSET_LIMIT:
+                    limit+=1
+                else:
```

Comment:
@huzaifmalik786 looks like offset and limit have been switched ?

limit will increment by limit = limit + FLOW_RUN_LOGS_OFFSET_LIMIT

Reply:
and we are not incrementing offset . logic is a bit off.